elide-core/src/main/java/com/yahoo/elide/graphql/PersistentResourceFetcher.java

```diff
@@ -157,46 +153,35 @@ private Object fetchObjects(Environment context) {
      */
     private Class getRecordType(RequestScope requestScope, GraphQLType outputType) {
         GraphQLObjectType graphQLType = (GraphQLObjectType) ((GraphQLList) outputType).getWrappedType();
+        String entityName = graphQLType.getName();
```

Comment:
Hmm, this was hacky. I changed it to fetch the `entityClass` using just the field name provided by graphql under `field.getName()`. Makes it more concise.